diff --git a/gulpfile.js b/gulpfile.js
index 0e17ab9..2807c88 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -34,7 +34,7 @@ app.addStyle = function(paths, outputFilename) {
 };
 
 app.addScript = function(paths, outputFilename) {
-    gulp.src(paths)
+    return gulp.src(paths)
         .pipe(plugins.plumber(function(error) {
             console.log(error.toString());
             this.emit('end');
@@ -106,10 +106,14 @@ gulp.task('styles', function() {
 });
 
 gulp.task('scripts', function() {
-    app.addScript([
+    var pipeline = new Pipeline();
+
+    pipeline.add([
         config.bowerDir+'/jquery/dist/jquery.js',
         config.assetsDir+'/js/main.js'
     ], 'site.js');
+
+    pipeline.run(app.addScript);
 });
 
 gulp.task('fonts', function() {
