diff --git a/gulpfile.js b/gulpfile.js
index a27fcbe..0e17ab9 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -14,7 +14,7 @@ var config = {
 var app = {};
 
 app.addStyle = function(paths, outputFilename) {
-    return gulp.src(paths).on('end', function() { console.log('start '+outputFilename)})
+    return gulp.src(paths)
         .pipe(plugins.plumber(function(error) {
             console.log(error.toString());
             this.emit('end');
@@ -30,7 +30,7 @@ app.addStyle = function(paths, outputFilename) {
         .pipe(plugins.rev.manifest(config.revManifestPath, {
             merge: true
         }))
-        .pipe(gulp.dest('.')).on('end', function() { console.log('end '+outputFilename)})
+        .pipe(gulp.dest('.'));
 };
 
 app.addScript = function(paths, outputFilename) {
