diff --git a/gulpfile.js b/gulpfile.js
index 4fb3d67..ed05b44 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -13,7 +13,7 @@ var config = {
 var app = {};
 
 app.addStyle = function(paths, outputFilename) {
-    gulp.src(paths).on('end', function() { console.log('start '+outputFilename)})
+    return gulp.src(paths).on('end', function() { console.log('start '+outputFilename)})
         .pipe(plugins.plumber())
         .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.init()))
         .pipe(plugins.sass())
@@ -56,11 +56,11 @@ gulp.task('styles', function() {
         config.bowerDir+'/font-awesome/css/font-awesome.css',
         config.assetsDir+'/sass/layout.scss',
         config.assetsDir+'/sass/styles.scss'
-    ], 'main.css');
-
-    app.addStyle([
-        config.assetsDir+'/sass/dinosaur.scss'
-    ], 'dinosaur.css');
+    ], 'main.css').on('end', function() {
+        app.addStyle([
+            config.assetsDir+'/sass/dinosaur.scss'
+        ], 'dinosaur.css');
+    });
 });
 
 gulp.task('scripts', function() {
