diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index c7b3b94..272eba4 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -9,8 +9,7 @@
         {% block stylesheets %}
             <link rel="stylesheet" href="{{ asset('vendor/bootstrap/dist/css/bootstrap.css') }}"/>
 
-            <link rel="stylesheet" href="{{ asset('css/styles.css') }}"/>
-            <link rel="stylesheet" href="{{ asset('css/layout.css') }}"/>
+            <link rel="stylesheet" href="{{ asset('css/main.css') }}"/>
 
             <!-- Google Fonts -->
             <link href='http://fonts.googleapis.com/css?family=Shadows+Into+Light' rel='stylesheet' type='text/css'>
diff --git a/gulpfile.js b/gulpfile.js
index c169ba3..c271445 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -1,6 +1,7 @@
 var gulp = require('gulp');
 var sass = require('gulp-sass');
 var sourcemaps = require('gulp-sourcemaps');
+var concat = require('gulp-concat');
 
 var config = {
     assetsDir: 'app/Resources/assets',
@@ -11,6 +12,7 @@ gulp.task('sass', function() {
     gulp.src(config.assetsDir+'/'+config.sassPattern)
         .pipe(sourcemaps.init())
         .pipe(sass())
+        .pipe(concat('main.css'))
         .pipe(sourcemaps.write('.'))
         .pipe(gulp.dest('web/css'));
 });
diff --git a/package.json b/package.json
index 9879f2b..bcc1bdd 100644
--- a/package.json
+++ b/package.json
@@ -7,6 +7,7 @@
   },
   "devDependencies": {
     "gulp": "^3.8.11",
+    "gulp-concat": "^2.5.2",
     "gulp-sass": "^2.3.2",
     "gulp-sourcemaps": "^1.5.0"
   },
