diff --git a/gulpfile.js b/gulpfile.js
index de63d5e..dcbe199 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -24,6 +24,19 @@ app.addStyle = function(paths, outputFilename) {
         .pipe(gulp.dest('web/css'));
 };
 
+app.addScript = function(paths, outputFilename) {
+    gulp.src(paths)
+        .pipe(plugins.plumber(function(error) {
+            console.log(error.toString());
+            this.emit('end');
+        }))
+        .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.init()))
+        .pipe(plugins.concat(outputFilename))
+        .pipe(config.production ? plugins.uglify() : plugins.util.noop())
+        .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.write('.')))
+        .pipe(gulp.dest('web/js'));
+};
+
 gulp.task('styles', function() {
     app.addStyle([
         config.bowerDir+'/bootstrap/dist/css/bootstrap.css',
@@ -37,19 +50,10 @@ gulp.task('styles', function() {
 });
 
 gulp.task('scripts', function() {
-    gulp.src([
+    app.addScript([
         config.bowerDir+'/jquery/dist/jquery.js',
         config.assetsDir+'/js/main.js'
-    ])
-        .pipe(plugins.plumber(function(error) {
-            console.log(error.toString());
-            this.emit('end');
-        }))
-        .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.init()))
-        .pipe(plugins.concat('site.js'))
-        .pipe(config.production ? plugins.uglify() : plugins.util.noop())
-        .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.write('.')))
-        .pipe(gulp.dest('web/js'));
+    ], 'site.js');
 });
 
 gulp.task('watch', function() {
