diff --git a/gulpfile.js b/gulpfile.js
index c271445..0b89326 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -2,6 +2,7 @@ var gulp = require('gulp');
 var sass = require('gulp-sass');
 var sourcemaps = require('gulp-sourcemaps');
 var concat = require('gulp-concat');
+var minifyCSS = require('gulp-minify-css');
 
 var config = {
     assetsDir: 'app/Resources/assets',
@@ -13,6 +14,7 @@ gulp.task('sass', function() {
         .pipe(sourcemaps.init())
         .pipe(sass())
         .pipe(concat('main.css'))
+        .pipe(minifyCSS())
         .pipe(sourcemaps.write('.'))
         .pipe(gulp.dest('web/css'));
 });
diff --git a/package.json b/package.json
index bcc1bdd..980a87b 100644
--- a/package.json
+++ b/package.json
@@ -8,6 +8,7 @@
   "devDependencies": {
     "gulp": "^3.8.11",
     "gulp-concat": "^2.5.2",
+    "gulp-minify-css": "^0.5.1",
     "gulp-sass": "^2.3.2",
     "gulp-sourcemaps": "^1.5.0"
   },
