diff --git a/gulpfile.js b/gulpfile.js
index 73ffbf8..927a8d4 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -7,17 +7,16 @@ var util = require('gulp-util');
 
 var config = {
     assetsDir: 'app/Resources/assets',
-    sassPattern: 'sass/**/*.scss'
+    sassPattern: 'sass/**/*.scss',
+    production: !!util.env.production
 };
 
-console.log(util.env.production);
-
 gulp.task('sass', function() {
     gulp.src(config.assetsDir+'/'+config.sassPattern)
         .pipe(sourcemaps.init())
         .pipe(sass())
         .pipe(concat('main.css'))
-        .pipe(minifyCSS())
+        .pipe(config.production ? minifyCSS() : util.noop())
         .pipe(sourcemaps.write('.'))
         .pipe(gulp.dest('web/css'));
 });
