diff --git a/app/Resources/assets/sass/styles.scss b/app/Resources/assets/sass/styles.scss
index a6481c7..aed51b3 100644
--- a/app/Resources/assets/sass/styles.scss
+++ b/app/Resources/assets/sass/styles.scss
@@ -23,10 +23,6 @@ $font-font: 'Shadows Into Light', cursive;
     text-align: center;
 }
 
-.dino-img-show {
-    margin-bottom: 50px;
-}
-
 /**Dino Show Page**/
 .btn-margin {
     margin-bottom: 50px;
diff --git a/app/Resources/views/dinosaurs/show.html.twig b/app/Resources/views/dinosaurs/show.html.twig
index ba9a9ae..658905f 100644
--- a/app/Resources/views/dinosaurs/show.html.twig
+++ b/app/Resources/views/dinosaurs/show.html.twig
@@ -1,5 +1,11 @@
 {% extends 'base.html.twig' %}
 
+{% block stylesheets %}
+    {{ parent() }}
+
+    <link rel="stylesheet" href="{{ asset('css/dinosaur.css') }}"/>
+{% endblock %}
+
 {% block body %}
     <div class="dino-header text-center">
         <h1>{{ dino.name }} the {{ dino.type }}</h1>
diff --git a/gulpfile.js b/gulpfile.js
index 17e7699..c8ff245 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -7,22 +7,31 @@ var config = {
     production: !!plugins.util.env.production,
     sourceMaps: !plugins.util.env.production
 };
+var app = {};
 
-gulp.task('sass', function() {
-    gulp.src([
-        config.assetsDir+'/sass/layout.scss',
-        config.assetsDir+'/sass/styles.scss'
-    ])
+app.addStyle = function(paths, outputFilename) {
+    gulp.src(paths)
         .pipe(plugins.plumber(function(error) {
             console.log(error.toString());
             this.emit('end');
         }))
         .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.init()))
         .pipe(plugins.sass())
-        .pipe(plugins.concat('main.css'))
+        .pipe(plugins.concat(outputFilename))
         .pipe(config.production ? plugins.minifyCss() : plugins.util.noop())
         .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.write('.')))
         .pipe(gulp.dest('web/css'));
+};
+
+gulp.task('sass', function() {
+    app.addStyle([
+        config.assetsDir+'/sass/layout.scss',
+        config.assetsDir+'/sass/styles.scss'
+    ], 'main.css');
+
+    app.addStyle([
+        config.assetsDir+'/sass/dinosaur.scss'
+    ], 'dinosaur.css');
 });
 
 gulp.task('watch', function() {
