diff --git a/gulpfile.js b/gulpfile.js
index 3469a6b..595f172 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -1,25 +1,20 @@
 var gulp = require('gulp');
-var sass = require('gulp-sass');
-var sourcemaps = require('gulp-sourcemaps');
-var concat = require('gulp-concat');
-var minifyCSS = require('gulp-minify-css');
-var util = require('gulp-util');
-var gulpif = require('gulp-if');
+var plugins = require('gulp-load-plugins')();
 
 var config = {
     assetsDir: 'app/Resources/assets',
     sassPattern: 'sass/**/*.scss',
-    production: !!util.env.production,
-    sourceMaps: !util.env.production
+    production: !!plugins.util.env.production,
+    sourceMaps: !plugins.util.env.production
 };
 
 gulp.task('sass', function() {
     gulp.src(config.assetsDir+'/'+config.sassPattern)
-        .pipe(gulpif(config.sourceMaps, sourcemaps.init()))
-        .pipe(sass())
-        .pipe(concat('main.css'))
-        .pipe(config.production ? minifyCSS() : util.noop())
-        .pipe(gulpif(config.sourceMaps, sourcemaps.write('.')))
+        .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.init()))
+        .pipe(plugins.sass())
+        .pipe(plugins.concat('main.css'))
+        .pipe(config.production ? plugins.minifyCSS() : plugins.util.noop())
+        .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.write('.')))
         .pipe(gulp.dest('web/css'));
 });
 
diff --git a/package.json b/package.json
index 40a64d6..2783d20 100644
--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
     "gulp": "^3.8.11",
     "gulp-concat": "^2.5.2",
     "gulp-if": "^1.2.5",
+    "gulp-load-plugins": "^0.8.1",
     "gulp-minify-css": "^0.5.1",
     "gulp-sass": "^1.3.3",
     "gulp-sourcemaps": "^1.5.0",
