diff --git a/gulpfile.js b/gulpfile.js
index 01c017a..06d818c 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -10,6 +10,10 @@ var config = {
 
 gulp.task('sass', function() {
     gulp.src(config.assetsDir+'/'+config.sassPattern)
+        .pipe(plugins.plumber(function(error) {
+            console.log(error.toString());
+            this.emit('end');
+        }))
         .pipe(plugins.if(config.sourceMaps, plugins.sourcemaps.init()))
         .pipe(plugins.sass())
         .pipe(plugins.concat('main.css'))
diff --git a/package.json b/package.json
index 3ee7037..8397caa 100644
--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
     "gulp-if": "^1.2.5",
     "gulp-load-plugins": "^0.8.1",
     "gulp-minify-css": "^0.5.1",
+    "gulp-plumber": "^1.0.0",
     "gulp-sass": "^2.3.2",
     "gulp-sourcemaps": "^1.5.0",
     "gulp-util": "^3.0.4"
