diff --git a/gulpfile.js b/gulpfile.js
index b5357ed..4bc3817 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -2,10 +2,12 @@ var gulp = require('gulp');
 var sass = require('gulp-sass');
 var sourcemaps = require('gulp-sourcemaps');
 
-gulp.task('default', function() {
+gulp.task('sass', function() {
     gulp.src('app/Resources/assets/sass/**/*.scss')
         .pipe(sourcemaps.init())
         .pipe(sass())
         .pipe(sourcemaps.write('.'))
         .pipe(gulp.dest('web/css'));
 });
+
+gulp.task('default', ['sass']);
