diff --git a/gulpfile.js b/gulpfile.js
index 4478c6f..b4e595d 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -21,7 +21,7 @@ app.addStyle = function(paths, outputFilename) {
         .pipe(gulp.dest('web/css'));
 };
 
-gulp.task('sass', function() {
+gulp.task('styles', function() {
     app.addStyle([
         config.bowerDir+'/bootstrap/dist/css/bootstrap.css',
         config.assetsDir+'/sass/layout.scss',
@@ -34,7 +34,7 @@ gulp.task('sass', function() {
 });
 
 gulp.task('watch', function() {
-    gulp.watch(config.assetsDir+'/'+config.sassPattern, ['sass'])
+    gulp.watch(config.assetsDir+'/'+config.sassPattern, ['styles'])
 });
 
-gulp.task('default', ['sass', 'watch']);
+gulp.task('default', ['styles', 'watch']);
